### ISSUE TITLE
my hm_loss in val is very large and won't decrease, can you give me some advice?

### ISSUE THREAD

**Issue by xiaosibai** [xingyizhou/CenterNet#312]
Those are the log and opt files  in my training work, can you find the mistake I made? I will appreciate it if you can help me.
[log.txt](https://github.com/xingyizhou/CenterNet/files/3562020/log.txt)
[opt.txt](https://github.com/xingyizhou/CenterNet/files/3562021/opt.txt)

**Comment by xingyizhou** [comment]
`--lr 5e-4` is for batchsize 128. You will need to use lr=5e-4/128*16 if you want to use batchsize 16, following the linear learning rate rule.
